source/command-monitoring/command-monitoring.rst

```diff
@@ -210,11 +139,11 @@ replies are received. As such, it would be innappropiate at this level for a dri
 custom logic around particular commands to determine what failure or success means for a particular
 command. Implementators of the API are free to handle these events as they see fit, which may include
 code that futher interprets replies to specific commands based on the presence or absence of other
-fields in the reply beyond the ‘ok’ field.
+fields in the reply beyond the ``ok`` field.
 
 *3. Why are document sequences included as BSON arrays?*
 
-The OP_MSG wire protocol was introduced in MongoDB 3.6, with document sequences as an optimization for bulk writes. The same optimization will be introduced for "find" and "aggregate" replies in 3.8. We have chosen to represent these OP_MSGs as single command or reply documents for now, until a need for a more accurate (and perhaps better-performing) command monitoring API for document sequences has been demonstrated.
```

Comment:
3.8 never happened and I wasn't sure if/when the upcoming work described here actually happened (the OP_MSG spec only mentions the bulk use case and I couldn't find tickets about this). I took out the sentence for now, but if anyone knows if/when this did happen happy to add back mention of it

Reply:
It didn't happen, at least not in drivers.  I recall doing some experiments to see if it was any faster and it wasn't, so we dropped it

Reply:
I realized that a section above also discussed the use of document sequences in find/aggregate replies so I have removed that as well.